use_cases/examples/home_loan_delinquency/dbt/Dockerfile

```diff
@@ -25,5 +25,5 @@ COPY --chown=dbt . $DBT_DIR
 # Run DBT debs to install dependencies
 RUN dbt deps
 
-# Run dbt
-ENTRYPOINT [ "dbt", "run", "--profiles-dir", "profiles" ]
+# Run dbt. Must specify whether test or run as a parameter (not baked in the entrypoint)
```

Comment:
We lose the --profiles-dir profiles then. Does that matter?

Maybe we should add export DBT_PROFILES_DIR=/dbt/profiles (and test it).

Reply:
great idea, much better than specifying the path at every command